vizro-core/examples/_chart-gallery/pages/_factories.py

```diff
@@ -74,3 +74,39 @@ def butterfly_factory(group: str):
             make_code_clipboard_from_py_file("butterfly.py"),
         ],
     )
+
+
+def connected_scatter_factory(group: str):
+    """Reusable function to create the page content for the column chart with a unique ID."""
+    return vm.Page(
+        id=f"{group}-connected-scatter",
+        path=f"{group}/connected-scatter",
+        title="Connected Scatter",
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+                #### What is a connected scatter chart?
+
+                A connected scatter chart visualizes two variables using dots, with lines connecting the dots in the
+                order of the data points. One variable is plotted along the x-axis and the other along the y-axis,
+                showing both the relationship and the sequence of the data.
+
+                &nbsp;
+
+                #### When should I use it?
+
+                Use connected scatter charts to show the relationship between two variables and the sequence of data
+                points. They are ideal for paired numerical data, helping to reveal trends and patterns over time or in
+                a specific order. Remember, correlation is not causation, so ensure your audience understands this to
+                avoid misinterpretations.
```

Comment:
```suggestion
                avoid misinterpretation.
```

Reply:
Or you could use the word "misrepresentation" I think, but misinterpretation (by the audience) is fine. Maybe you could put a link under "correlation is not causation" to point readers to further info, but I reckon that's unnecessary for all but the most naive reader.